mmdet3d/structures/bbox_3d/base_box3d.py

```diff
@@ -463,11 +463,17 @@ def overlaps(cls, boxes1, boxes2, mode='iou'):
         # height overlap
         overlaps_h = cls.height_overlaps(boxes1, boxes2)
 
+        # Restrict the min values of W and H to avoid memory overflow in
+        # ``box_iou_rotated``.
+        boxes1_bev, boxes2_bev = boxes1.bev, boxes2.bev
+        boxes1_bev[:, 2:4] = boxes1_bev[:, 2:4].clamp_min(1e-4)
```

Comment:
clamp_min -> clamp(min=1e-4) is more common. Not sure whether clamp_min has full support of different pytorch versions.